api.yaml

```diff
@@ -291,7 +291,21 @@ paths:
           description: Donor account features enabled
         '400':
           description: Invalid data or already enabled
-
+  /people/{personId}/regular-giving:
```

Comment:
I think this needs a param def to by valid syntax:

> Declared path parameter "personId" needs to be defined as a path parameter at either the path or operation level

Slightly frustratingly it seems this has to be declared repetitively for each path and can't be `$ref`'d to a shared definition.